src/rhsmlib/services/entitlement/merged_pools.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import print_function, division, absolute_import
```

Comment:
This should just stay in `src/subscription_manager/managerlib.py` and be imported.